sklearn/grid_search.py

```diff
@@ -111,6 +113,47 @@ def __len__(self):
         return sum(product(len(v) for v in p.values()) if p else 1
                    for p in self.param_grid)
 
+    def __getitem__(self, ind):
+        """Get the parameters that would be ``ind``th in iteration
+
+        Parameters
+        ----------
+        ind : int
+            The iteration index
+
+        Returns
+        -------
+        params : dict of string to any
```

Comment:
The same description is used above in object constructor. It means "a dict mapping strings to arbitrary-type values".